### PR TITLE
support m4a file extension

### PR DESCRIPTION
**Pull request by VICrazy**
### :sparkles: What kind of change does this PR introduce? (Bug fix, feature, docs update...)

feature
### :arrow_heading_down: What is the current behavior?

The m4a audio file extension is not supported now.
### :new: What is the new behavior (if this is a feature change)?

The m4a audio file extension are already supported.
### :boom: Does this PR introduce a breaking change?

no
### :bug: Recommendations for testing

Download the m4a audio file via http, and then play the file
### :memo: Links to relevant issues/docs

https://github.com/Baseflow/flutter_cache_manager/pull/172/files
### :thinking: Checklist before submitting

- [x] All projects build
- [x] Follows style guide lines ([code style guide](https://github.com/Baseflow/flutter_cache_manager/blob/develop/CONTRIBUTING.md))
- [x] Relevant documentation was updated
- [x] Rebased onto current develop

### CODE REVIEW

File: flutter_cache_manager/lib/src/web/mime_converter.dart [hunk 1]
```diff
@@ -54,6 +54,7 @@ const mimeTypes = {
   'audio/aac': '.aac',
   'audio/x-aac': '.aac',
   'audio/midi audio/x-midi': '.midi',
+  'audio/x-m4a': '.m4a',
```
Comment (renefloor) [inline flutter_cache_manager/lib/src/web/mime_converter.dart:57]: [Looking at this list](http://help.dottoro.com/lapuadlp.php), the main type seems to me `audio/m4a` and x-m4a is an alias for that. I think it is better to include both.
```suggestion
  'audio/x-m4a': '.m4a',
  'audio/m4a': '.m4a',
```

Comment (VICrazy) [reply flutter_cache_manager/lib/src/web/mime_converter.dart:57]: OK

**Comment by renefloor** [comment]
Sorry about the analyzer, yours is the first PR since the new Flutter stable. If you want I can create a separate PR to fix the analyzer issues.

**Comment by VICrazy** [comment]
> Sorry about the analyzer, yours is the first PR since the new Flutter stable. If you want I can create a separate PR to fix the analyzer issues.

Thanks, but I want to try fix it.

**Pull request merged**